cypress/e2e/nonParallel/data-import/e2e/mrc-import-file-for-update-items.cy.js

```diff
@@ -364,7 +364,7 @@ describe('ui-data-import', () => {
     NewFieldMappingProfile.fillMaterialType('"electronic resource"');
     NewFieldMappingProfile.addItemNotes('"Electronic bookplate"', '"Smith Family Foundation"', 'Mark for all affected records');
     NewFieldMappingProfile.fillPermanentLoanType('"Can circulate"');
-    NewFieldMappingProfile.fillStatus('"Available"');
+    NewFieldMappingProfile.fillStatus('Available');
```

Comment:
use ITEM_STATUSES.AVAILABLE, LOAN_TYPE_NAMES.CAN_CIRCULATE, MATERIAL_TYPE_NAMES.BOOK

ypu can add material type: electronic resource to constants

Reply:
the same